apps/web/src/schema/identity/role-binding/api-verbs/list.ts

```diff
@@ -1,12 +1,18 @@
 import type { Query } from '@cloudforet/core-lib/space-connector/type';
 
+import type { RoleBindingModel } from '@/schema/identity/role-binding/model';
 import type { RoleType } from '@/schema/identity/role/type';
 
-export interface RoleListParameters {
+export interface RoleBindingListParameters {
     query?: Query;
     role_binding_id?: string;
     user_id?: string;
     role_id?: string;
     role_type?: RoleType;
     workspace_id?: string;
 }
+
+export interface RoleBindingListResponse {
+    results: RoleBindingModel[];
+    total_count: number;
+}
```

Comment:
Suggestion: How about utilizing `ListResponse` type?
ListResponse looks like the following:
`export interface ListResponse<T> {
    results?: T[];
    total_count?: number;
}`

Reply:
I agree with Nayeong. Also, in identity V2, if there is no value, an `{}` is returned as the resonse.